src/app/courses/courses-list/courses-list.component.html

```diff
@@ -1,7 +1,8 @@
 <ul class="coursesList">
-    <li *ngFor="let item of courses; trackBy: trackByFn">
+    <div class="noCourses" *ngIf="courses.length === 0">No data feel free to enter new course</div>
```

Comment:
yes it will not work. The div will be shown only when there is not any courses in the whole list, but not after the filter. I though it should work in this way.

Reply:
Yes, you are right, thanks - I misunderstood the requirements for this HW)